code/modules/client/preference/link_processing.dm

```diff
@@ -1068,7 +1068,121 @@
 				if("edit_2fa")
 					// Do this async so we arent holding up a topic() call
 					INVOKE_ASYNC(user.client, /client.proc/edit_2fa)
-					return // We return here to avoid focus being lost
+					return // We return here to avoid focus being load_strings_file
+
+				if("keybindings")
+					if(!keybindings_overrides)
+						keybindings_overrides = list()
+
+					if(href_list["set"])
+						var/datum/keybinding/KB = locateUID(href_list["set"])
+						if(KB)
+							if(href_list["key"])
+								var/old_key = href_list["old"]
+								var/new_key = copytext(url_decode(href_list["key"]), 1, 16)
+								var/alt_mod = text2num(href_list["alt"]) ? "Alt" : ""
+								var/ctrl_mod = text2num(href_list["ctrl"]) ? "Ctrl" : ""
+								var/shift_mod = text2num(href_list["shift"]) ? "Shift" : ""
+								var/numpad = (text2num(href_list["numpad"]) && text2num(new_key)) ? "Numpad" : ""
+								var/clear = text2num(href_list["clear_key"])
+
+								if(!(length_char(new_key) == 1 && text2ascii(new_key) >= 0x80)) // Don't uppercase unicode stuff
+									new_key = uppertext(new_key)
+
+								// Map for JS keys
+								var/static/list/key_map = list(
+									"UP" = "North",
+									"RIGHT" = "East",
+									"DOWN" = "South",
+									"LEFT" = "West",
+									"INSERT" = "Insert",
+									"HOME" = "Northwest",
+									"PAGEUP" = "Northeast",
+									"DEL" = "Delete",
+									"END" = "Southwest",
+									"PAGEDOWN" = "Southeast",
+									"SPACEBAR" = "Space",
```

Comment:
These being cardinals is laziness for the numpad bindings of how `Home` is in the northwest on a numpad. There must be better BYOND binds for this.

Reply:
This just determines their name in the menu, I agree however that they should be tied to the actual key name. I will change it now. As for what they are bound to, it can just be re-bound so 🤷🏼‍♂️